src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+    private static final String LOGIN = "bob";
+    private static final char [] PASSWORD = { '1','2','3','4' };
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login(LOGIN, String.valueOf(PASSWORD));
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't identification with login " + LOGIN, e);
```

Comment:
We can use formatting here:
`logger.error("Can't identification with login {}", LOGIN, e);`